### ISSUE TITLE
Homework Assignment

### ISSUE THREAD

**Issue by hieule1607** [hieule1607/codershool#1]
Here is my completed homework assignment! Thank you.

/cc @coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Swift submission, hieule1607!
:laughing: Everything looks good with the format of your submission. We'll be providing a detailed review soon!

**Comment by coderschoolreview** [comment]
### Thank you for your Swift submission, hieule1607!

 :grin: Everything looks good with the format of your submission. We'll be reviewing your submission soon!

**Comment by coderschoolreview** [comment]
:+1: Nice job with the Tip Calculator.

### Some small comments:
- Good job with the animation :open_mouth:
- Good usage of the NSUserDefault but you should set the saving time for the bill amount as the the requirement ask.
- When users change the default percentage in `Settings`, the chosen percentage in `Home` is changed, it's right. But on the contrary, when users change in `Home`, the chosen percentage in `Settings` shouldn't be changed.
- Your app looks a little dark. It'd be better if you make it brighter :smile: 

Congratulation on completing the prework! The result will be emailed you next week.

**Comment by hieule1607** [comment]
Thank for your comments. I will learn from this comments. :)

Vào Thứ Bảy, ngày 20 tháng 2 năm 2016, coderschoolreview <
notifications@github.com> đã viết:

**Comment by harley** [comment]
Hi @hieule1607 -- we sent you an email a few days ago but haven't heard your confirmation. Do you see our email?

**Comment by hieule1607** [comment]
Yes, I see your email. I forgot to join the FB group. Now, I'm going to
join the FB group. Sorry about that.
